whois-commons/src/main/resources/applicationContext-commons.xml

```diff
@@ -17,11 +17,11 @@
     <!-- @Transactional CAN be used on private methods with mode="aspectj". However, annotations on interfaces will NOT work! -->
     <tx:annotation-driven mode="aspectj"/>
```

Comment:
Also according to: http://www.springframework.org/schema/tx/spring-tx.xsd
We can replace the `tx:annotation-driven` element:
```
	See org.springframework.transaction.annotation.EnableTransactionManagement Javadoc
	for information on code-based alternatives to this XML element.
```

Reply:
We should be careful as there are multiple tranbsaction in whois, I remeber having seperate one for nrtmv4

Reply:
I only see one `<tx:annotation-driven>` element in Whois now, maybe we fixed that?

Reply:
We specify @Transactional(transactionManager = "nrtmTransactionManager") manually on reposiritry layer so all good

Reply:
Done